tensilelite/Tensile/Components/LSU.py

```diff
@@ -312,11 +328,20 @@ def writeReadReduction(self, writer, kernel):
                     for r in range(0, kernel["LocalSplitU"]):
                         regIdx = (i * numInstPerVW + v) * regsPerStore
                         offset = r * ldsStride + regIdx * (bpr * kernel["WavefrontSize"])
+                        srcvgpr = vgpr(addr)
```

Comment:
we can simplify this like following.
srcvgpr = vgpr(addr + %d)%(offset//65536)

Reply:
No need to check Has160KLDS here. Use nakajee's suggestion.